### ISSUE TITLE
About the Stereo Selective Whitening Loss.

### ISSUE THREAD

**Issue by Zhaohuai-L** [jiaw-z/FeatureConsistentStereo#8]
Hi , Thanks a lot for your great work,I mitigated SSW and SCF to my own Project with similar backbone. But the SSW loss is a NAN, so I debuged it and  found it's 'num_sensitive_sum' being  zero after ‘mask_matrix’ multiplied by  self.reversal_eye.
Even if I set 'num_sensitive_sum' to 0.0001 , the SWW loss becomes zero too.

Is there a bug here or am I understanding it wrong?

```            mask_matrix = torch.zeros(B, self.dim, self.dim).cuda()
            mask_matrix = mask_matrix.view(B, -1)
            for midx in range(B):
                mask_matrix[midx][indices] = 1
            mask_matrix = mask_matrix.view(B, self.dim, self.dim)
            mask_matrix = mask_matrix * self.reversal_eye
            num_sensitive_sum = torch.sum(mask_matrix)
            if num_sensitive_sum==0:
                num_sensitive_sum=0.0001
```

**Comment by jiaw-z** [comment]
Sorry to reply late. num_sensitive_sum sums the number of pixels that are sensitive to viewpoint changes. In our experiments, we did not observe cases in that  num_sensitive_sum equals zero. And it is not proper to set num_sensitive_sum to 0.0001 as it is an integer.

**Comment by Zhaohuai-L** [comment]
Ok, thanks for your reply.
